include/picongpu/param/transitionRadiation.param

```diff
@@ -0,0 +1,175 @@
+/* Copyright 2013-2019 Rene Widera, Richard Pausch, Finn-Ole Carstens
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *
+ * Definition of frequency space, number of observers, filters, form factors and
+ * window functions of the radiation plugin.
+ *
+ * All values set here determine what the radiation plugin will compute.
+ * The observation direction is defined in a seperate file `radiationObserver.param`.
+ * On the comand line the plugin still needs to be called for each species the radiation
+ * should be computed for.
+ */
+#pragma once
+
+/**
+ * radiation verbose level:
+ * 0=nothing, 1=physics, 2=simulation_state, 4=memory, 8=critical
+ */
+#define PIC_VERBOSE_RADIATION 3
+
+#include "picongpu/plugins/radiation/debug/PIConGPUVerboseLogRadiation.hpp"
+#include "picongpu/algorithms/Gamma.def"
+#include "picongpu/traits/attribute/GetMass.hpp"
+#include "picongpu/particles/manipulators/manipulators.def"
+
+namespace picongpu
+{
+namespace transitionRadiation
+{
+namespace linearFrequencies
+{
+    namespace SI
+    {
+        //! mimimum frequency of the linear frequency scale in units of [1/s]
+        constexpr float_64 omegaMin = 0.0;
+        //! maximum frequency of the linear frequency scale in units of [1/s]
+        constexpr float_64 omegaMax = 1.06e16;
+    }
+
+    //! number of frequency values to compute in the linear frequency [unitless]
+    constexpr unsigned int nOmega = 512;
+
+} // namespace linearFrequencies
+
+namespace logFrequencies
+{
+    namespace SI
+    {
+        //! mimimum frequency of the logarithmic frequency scale in units of [1/s]
+        constexpr float_64 omegaMin = 1.0e13;
+        //! maximum frequency of the logarithmic frequency scale in units of [1/s]
+        constexpr float_64 omegaMax = 1.0e17;
+    }
+
+    //! number of frequency values to compute in the logarithmic frequency [unitless] 
+    constexpr unsigned int nOmega = 256;
+
+} // namespace logFrequencies
+
+
+namespace listFrequencies
+{
+    //! path to text file with frequencies
+    constexpr char listLocation[] = "/path/to/frequency_list";
```

Comment:
Have you tested this?

Reply:
Nope. I included them because they are included in the radiation plugin and maybe people find a use for it. The differences from the list frequencies and the log/lin frequencies are not very big, so I'd expect this to work, if it works in the _radiation plugin_.

Reply:
I am fine with that.

Reply:
It did not work correctly in the radiation plugin - see #3053

Reply:
Should I note this somewhere? I tried to leave everything frequency related similar to how it is currently implemented in the radiation plugin. This way one can just copy+paste the changes from the reworked radiation plugin into the transition radiation plugin.